ironfish/src/testUtilities/nodeTest.ts

```diff
@@ -1,6 +1,7 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import './matchers'
```

Comment:
So all of our matchers are available in any test. This is an orthagonal change but I snuck it in here.